HelpSource/Classes/Event.schelp

```diff
@@ -240,6 +240,22 @@ q.y = q.x * q.n;	// shorter way to do the same (pseudo-methods)
 q.y;			// [100, 200, 300]
 ::
 
+subsection::Syntax subtleties
+
+Events have a very special meaning in a musical context within SuperCollider, especially when it comes to sequencing with Patterns. Nevertheless, as the class inherits from link::Classes/IdentityDictionary::, it is often used as a convenient-to-type replacement for (Identity)Dictionary-like objects. The syntax very much reminds of the declaration of keyword arguments in function or method calls. However, there is a subtle difference:
+
+code::
+e = (a: 10);  // -> ('a': 10)
+f = (a : 10); // -> nil
+              // ERROR: Primitive '_IdentDict_Put' failed.
+              // a is treated as a variable!
```

Comment:
Can you add `a = nil; // make sure a is nil` beforehand just to make sure it actually is not assigned to anything? There's a= assignment earlier in this help doc.

Reply:
Yes, will do tomorrow. It's already quite late and I need a little sleep.